pkgs/applications/networking/instant-messengers/sky/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv, fetchurl, file, lib, libX11, libXScrnSaver
+, gcc-unwrapped, libGL, qt5, SDL, libpulseaudio
+, libXrandr, libXext, libXcursor, libXinerama, libXi
+, ffmpeg-full, curl, sqlite, openssl_1_1_0
+, libuuid, openh264, libv4l, libxkbfile, libXv, zlib, libXmu
+, libXtst, libXdamage, pam, patchelfUnstable, libXfixes, libXrender, libjpeg_original
+}:
+ let
+   # Sky is linked to the libjpeg 8 version and checks for the version number in the code.
+   libjpeg_original_fix = libjpeg_original.overrideAttrs (oldAttrs: {
+    src = fetchurl{
+      url = http://www.ijg.org/files/jpegsrc.v8d.tar.gz;
+      sha256 = "1cz0dy05mgxqdgjf52p54yxpyy95rgl30cnazdrfmw7hfca9n0h0";
+    };
+  });
+in
+stdenv.mkDerivation rec {
+  version = "2.1.7300.1";
+  name = "sky-${version}";
+  src = fetchurl {
+    url = "https://tel.red/repos/archlinux/sky-archlinux-${(lib.concatStringsSep "." (lib.take 3 (lib.splitString "." version))) + "-" + (lib.last (lib.splitString "." version))}-x86_64.pkg.tar.xz";
+    sha256 = "1423d4c091faa56fcda5ccf53c87a151a8fe7f1cc8cda748c4198cb51f7b7143";
+  };
+  buildInputs = [ 
+    stdenv
+    file
+    gcc-unwrapped
+    qt5.qtbase
+    SDL
+    ffmpeg-full
+    sqlite
+    openssl_1_1_0
+    openh264
+    pam
+    curl
+    libX11 libXScrnSaver libGL libpulseaudio libXrandr libXext libXcursor libXinerama libXi libuuid libv4l libxkbfile libXv zlib libXmu libXtst libXdamage libXfixes libXrender
```

Comment:
Could maybe use a line break or two here

Reply:
As a short question adding this led to  
> is not supported on ‘x86_64-unknown-linux-gnu’, refusing to evaluate
How would I work around it i could not find documentation on this